pom.xml

```diff
@@ -79,11 +79,5 @@
       <version>3.8.2</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>1.2.14</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
If log4j is being removed, then I think we should delete https://github.com/joelhockey/jacknji11/blob/master/src/test/resources/log4j.xml

Reply:
Thanks. Indeed. I removed it in the feature branch.